staff/almu-itur/cv/index.html

```diff
@@ -0,0 +1,78 @@
+index <!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>CV ALMUDENA ITURGOYEN</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
```

Comment:
In this case there is not css file, right?

Reply:
Correct! I'll delete the line. Thank you!